<launch>
  <rosparam command="load" file="$(find easy_camera_lidar_calibration)/config/config.yaml" />
  <param name="use_sim_time" value="true"/>

  <node name="lidar2pcd" pkg="easy_camera_lidar_calibration" type="lidar2pcd" output="screen"/>

</launch>